src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -65,8 +65,10 @@ const Body = BaseFormWithPolling.extend(
           url: getAuthenticatorUrl('probe'),
           // in loopback server, SSL handshake sometimes takes more than 100ms and thus needs additional timeout
           // however, increasing timeout is a temporary solution since user will need to wait much longer in worst case
-          // TODO: OKTA-278573 Android timeout is temporarily set to 3000ms and needs optimization post-Beta
-          timeout: BrowserFeatures.isAndroid() ? 3000 : 100
+          // TODO: OKTA-278573 Android timeout is temporarily set to 3000ms and needs optimization post-Beta.
```

Comment:
possible to reorganize all these comments into /** single comment **/

Reply:
Done! Tweaked comments for clarity.